system/API/ResponseTrait.php

```diff
@@ -53,39 +52,40 @@
  */
 trait ResponseTrait
 {
+
 	/**
 	 * Allows child classes to override the
 	 * status code that is used in their API.
 	 *
 	 * @var array
 	 */
 	protected $codes = [
-		'created'                   => 201,
-		'deleted'                   => 200,
-		'invalid_request'           => 400,
+		'created' => 201,
```

Comment:
That is probably a setting in my IDE, which some sort of beautifier should pickup and fix for us.
Someone, I am sure, will submit a PR to align these, and then someone will submit a PR to unalign them, ad infinitum. I am comfortable waiting until we have a tool to take care of that for us, so long as we are generally ok.